docs/releasehistory.rst

```diff
@@ -1,6 +1,15 @@
 Release History
 ***************
 
+0.21.5 - Bugfix release
```

Comment:
```suggestion
0.22.0 - Bugfix release
```

I think we should make this bump-the-version-release since changing the integrator is kinda a big deal since it changes what someone would say the integrator they used in their method section was. I'm not sure if we should also just add a warning and keep the warning in for a release or two saying the integrator here has changed. 

I'm still going to approve the PR, but just thought we should have a discussion about how to make sure our users are prepared for this change.

Reply:
@jchodera let me know what you think. I know people should pin versions and this code is pre 1.x so people are using it at their own peril.

Reply:
Sounds good, do you think the release notes are enough to communicate the change? Or should we add a warning or something? Maybe it really isn't an issue at all since people should pin versions when generating data for papers.